digits/templates/models/images/classification/new.html

```diff
@@ -437,6 +438,17 @@ <h4 style="display:inline-block;">Python Layers</h4>
         $("#torch-warning").hide();
     $('#stdnetRole a[href="'+"#"+fwid+"_standard"+'"]').tab('show');
     $('#customFramework a[href="'+"#"+fwid+"_custom"+'"]').tab('show');
+
+    $('#iter-size-option').hide();
+    $('#iter_size').prop('disabled', true);
+    {% for fw in frameworks %}
+    {% if fw.can_accumulate_gradients() %}
+    if (fwid == "{{ fw.get_id() }}") {
+        $('#iter_size').prop('disabled', false);
+        $('#iter-size-option').show();
+    }
+    {% endif %}
+    {% endfor %}
```

Comment:
or maybe you can extend the structure on lines 414-416 with a `can_accumulate_gradients` field for a more direct access to framework properties?